docs/webactions.md

```diff
@@ -88,7 +88,7 @@ An OpenWhisk action that is not a web action requires both authentication and mu
 
 1. `headers`: a JSON object where the keys are header-names and the values are string values for those headers (default is no headers).
 2. `statusCode`: a valid HTTP status code (default is 200 OK).
-3. `body`: a string which is either plain text or a base64 encoded string (for binary data).
+3. `body`: a string which is either plain text or a base64 encoded string for binary data (default is empty response).
```

Comment:
Is this true?

if I `return ["statusCode": 405]`, I'll get a `text/plain` response with the body set to `A request was made of a resource using a request method not supported by that resource;`.

i.e. default is an empty response only in the case of a `200` status code.